### DISCUSSION TITLE
Regarding the behavior of the TST side panel opening automatically when the browser starts

### DISCUSSION THREAD

**Discussion by radiconkid** [piroor/treestyletab#3615] [Q&A]
I want to start with the side panel closed.
I would be happy if it would be possible to change the shortcut "F1" that displays TST.

**Comment by irvinm** [comment]
On a standard install (even without TST), if you load something in the sidebar and then restart the browser, that same item (History, Bookmarks, etc.) will start in the sidebar after the restart.  

Now the same is true with TST.  If you close Firefox with TST open, it will restart that way.  If you close Firefox with TST closed, it will restart that way.

You can already change the keyboard shortcut for TST:  

![image](https://github.com/user-attachments/assets/019d41fa-594d-4bf6-89a0-f80c95515213)

Are you asking for a way to close Firefox with the sidebar open and after a restart for the sidebar to be closed?

**Comment by radiconkid** [comment]
Thank you very much. I had forgotten that I could edit the extension shortcuts.
I see that the side panel display at startup is a Firefox specification; I want to use the Floorp browser, so I want to display TST within the Floorp side panel.
The reason I want to hide TST is because it would be duplicated if I show it at startup.
Thank you. I will see if there is another way.
![Screenshot from 2024-08-22 11-14-51](https://github.com/user-attachments/assets/cae27bfb-c693-4e80-a68b-3b3e8e91f669)
